multimedia/vapoursynth/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            vapoursynth vapoursynth 51 R
+
+categories              multimedia python
+platforms               darwin
+license                 LGPGL-2.1
+
+description             A video processing framework with simplicity in mind
+
+long_description        VapourSynth is an application for video manipulation. \
+                        Or a plugin. Or a library. It’s hard to tell because \
+                        it has a core library written in C++ and a Python \
+                        module to allow video scripts to be created. The \
+                        software has been heavily inspired by Avisynth.
+
+checksums               rmd160  95ad629c46c5310328ca466287a7e723ef7d07a0 \
+                        sha256  5e71167a0d8082bff2c11e2062c01348b49abfcf7ef5c982fa2a0702f3a654ed \
+                        size    618942
+
+maintainers             {gmail.com:herby.gillot @herbygillot} \
+                        openmaintainer
+
+depends_build-append    port:autoconf \
+                        port:automake \
+                        port:py38-cython
+
+depends_lib-append      port:ffmpeg \
+                        port:ImageMagick \
+                        port:libass \
+                        port:libiconv \
+                        port:pkgconfig \
+                        port:python38 \
+                        port:zimg
+
+use_autoreconf          yes
+autoreconf.cmd          ./autogen.sh
+autoreconf.args
```

Comment:
These autoreconf-related lines should be before the depends_build lines, since changing autoreconf.cmd will remove the autoconf and automake build dependencies.

Reply:
Thanks for the review, Ryan.  Yes, I learned this recently but hadn't gone back to fix prior port PRs.  Change made, added missing `libtool` requirement.